### DISCUSSION TITLE
Eslint configuration example

### DISCUSSION THREAD

**Discussion by ilyaDegtyarenko** [nuxt/nuxt#17382] [Questions]
Can someone provide a configuration example for composition-api?

**Comment by Diizzayy** [comment]
@ilyaDegtyarenko I'd suggest you use the [official eslint config for Nuxt ](https://github.com/nuxt/eslint-config#typescript).

You can run `npx nuxi init -t diizzayy/n3-starter nuxt-app` to create a new nuxt 3 app with ESLint preconfigured ( this is the repo - [Nuxt 3 Minimal Starter with ESLint](https://github.com/Diizzayy/n3-starter#nuxt-3-minimal-starter-with-eslint) )
